backend/fleet_management/serializers.py

```diff
@@ -193,13 +193,9 @@ def create(self, validated_data):
         except ObjectDoesNotExist as e:
             raise ValidationError(e.args[0])
 
-        total_cost = Money(
-            currency=data["total_cost.currency"], amount=data["total_cost.amount"]
-        )
-
```

Comment:
why are we removing this?

Reply:
The newest version of django-money by default serialize `MoneyField` to the two fields: `total_cost` and `total_cost_currency`, as you can see here https://github.com/CodeForPoznan/pah-fm/pull/610/files#diff-d7c04cd8b1db6e227ddf95748d6df2a1bb8ecc8f5c0254d69dc880d9a7acfed3R58 Using the new library behavior we can have less code.